common-moonmage.lic

```diff
@@ -320,4 +320,36 @@ module DRCMM
     DRC.message("Could not set planet data. Cannot cast #{data['abbrev']}")
   end
 
+  def set_moon_data(data, settings = nil)
```

Comment:
Are moon mages the only guild that needs this info? Do traders have spells that rely on specific moons?

Reply:
@KatoakDR should this comment hold up merging?

Reply:
Nope, was just curious!

Reply:
Can we please commit this now, because some spells are broken for moonies. And I'll clean up after and check the trader question? 

I need to make a trader.

Reply:
Katoak and I chatted on discord. Resolving.